extras.php

```diff
@@ -167,6 +167,12 @@ function rest_cookie_collect_status() {
  * @return string URL for the user's avatar, empty string otherwise.
  */
 function rest_get_avatar_url( $email ) {
+	/**
+	 * Use the WP Core `get_avatar_url()` function introduced in 4.2.
+	 */
+	if ( function_exists( 'get_avatar_url') ) {
```

Comment:
@WP-API/amigos what do you think about just bumping the minimum WP version here to 4.2?  Instead of checking that this function exists?